config/karma.conf.js

```diff
@@ -77,7 +77,7 @@ function configurator(config) {
     // available browser launchers:
     // https://npmjs.org/browse/keyword/karma-launcher
     // If run in travisCI, use firefox, otherwise, use Chrome.
```

Comment:
I didn't notice this before. Why do we use different browsers in different envs? Can we also have Chrome running on Travis?

Reply:
I guess it was to make it faster originally. I'll send a PR after this to also run Chrome on Travis.